composer.json

```diff
@@ -4,7 +4,7 @@
     "license": "GPL-2.0-or-later",
     "type": "drupal-module",
     "require": {
-        "dpc-sdp/tide_core": "^2.0.0"
+        "dpc-sdp/tide_core": "dev-feature/SDPA-5396-drupalcore-9 as 2.0.14"
```

Comment:
How long will this dev version be used?

Reply:
you meant ` "dpc-sdp/tide_core": "dev-feature/SDPA-5396-drupalcore-9 as 2.0.14"` ?
will remove before merging